doc/ref_model/chapters/chapter03.md

```diff
@@ -185,4 +185,30 @@ Networking, alongside Compute and Storage, is an integral part of the Cloud Infr
 
 Normalization of the integration reference points between different layers of the Cloud Infrastructure architecture is one of the main concerns. In the networking context the primary focus is directed on the packet flow and control flow interfaces between the virtual reosurces (referred to as Software (SW) Virtualisation Layer) and physical resources (referred to as Hardware (HW) Infrastructure Layer), as well as on related integration into the various MANO reference points (hardware/network infrastructure management, orchestration). The identification of these two different layers (SW Virtualisation Layer and HW Infrastructure Layer) remains in alignment with the separation of resources into virtual and physical resources, generally used in this document, see e.g. Figure 3-1. The importance of understanding the seperation of concerns between SW Virtualisation Layer and HW Infrastructure Layer is important because without it, the cardinality of having multiple CaaS and IaaS instances executing on their own private virtual resources from the single shared HW Infrastructure Layer cannot be expressed into separate administrative domains.
 
+<p align="center"><img src="../figures/RM-Chapter4-Network-Model.png" alt="Network Model" Title="Network Model" width="65%"/></p>
+<p align="center"><b>Figure 3-5:</b> Network Model</p>
 
+Principles that should be followed during development and definition of Networking scopes for Reference Model, Reference Architectures and down to Reference Implementations and Reference Conformance test suites under CNTT:
+
+• Abstraction: A standardized layer between Virtualisation Layer domains and the Networking Hardware Layer resource pool hiding the implementation details of the Networking Hardware.
```

Comment:
Suggest change to the following using terms previously defined in this chapter. Also, abstraction is a process:
"Abstraction: A standardized layer between the Virtualisation Layers and the Network Physical Resources Layer that hides (or abstracts) the details of the Network Physical resources from the Virtualisation Layers."

Reply:
@pgoyal01, text updated.  Please have a look whether your concerns are addressed.

Reply:
@walterkozlowski I think "domains" is redundant in "Virtualisation Layer domains"